docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -170,7 +172,7 @@ sudo /usr/local/bin/besu/bin/besu --data-path=/var/lib/besu --data-storage-forma
 
 ### Valid keypair not provided
 
-`java.lang.IllegalArgumentException: Supplied file does not contain valid keyPair pair.`
+- `java.lang.IllegalArgumentException: Supplied file does not contain valid keyPair pair.`
```

Comment:
ditto recommend concatenate does not > doesn't as per MS style

Reply:
same as above - in shell error message.